src/main/java/venus/Storage.java

```diff
@@ -0,0 +1,136 @@
+package venus;
+
+/**
+ * This is a Storage class that controls data to be stored or loaded from .txt data file.
+ *
+ * @author peterXGD
+ * @since 2024-02-05
+ */
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import java.time.format.DateTimeParseException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
+
+    /**
+     * Returns Storage that help store or load content from a filepath relative to the project
+     * structure.
+     *
+     * @param filePath Relative filepath of data in project.
+     */
+    public Storage (String filePath) {
+        File folder = new File (filePath.split(File.separator)[0]);
+        if (!folder.exists()) { // handling folder does not exist issues
+            folder.mkdir();
+            System.out.println("Folder does not exist, data folder is created");
+        }
+        if (!Files.exists(Path.of("data" + File.separator + "venus.txt"))) { // handling file does not exist
+            File f = new File("data" + File.separator + "venus.txt");
+        }
+        this.filePath = filePath;
+    }
+
+    /**
+     * Store a single line of task into file.
+     *
+     * @param textToAdd The toString() string of a task that is stored.
+     * @throws FileNotFoundException Throws exception when file cannot be found.
+     */
+    public void saveToFile(String textToAdd) throws FileNotFoundException {
+        try {
+            FileWriter fw = new FileWriter(filePath, true);
+            String amendedText = textToAdd.replaceAll("\\[\\s\\]", "|0|")
```

Comment:
The code for 'amendedText' is repeated in both the saveToFile and saveAllFile methods. You can consider extracting the logic into a separate method to improve code reusability.